scripts/build.js

```diff
@@ -207,8 +202,8 @@ if (manifest.declarative_net_request?.rule_resources) {
       resources: redirectResources.map((filename) =>
         join('rule_resources/redirects', filename),
       ),
-      all_frames: true,
```

Comment:
There is no `all_frames` key in the `web_accessible_resources` (https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/web_accessible_resources)